docs/source/guide.rst

```diff
@@ -346,9 +346,15 @@ Prioritizing Messages
 Say your app has some actors that are higher priority than others: for
 example, actors that affect your UI and make users wait, or are
 otherwise user-facing, versus actors that aren't.  When choosing
-between two concurrent messages to run, Dramatiq will run the Message
+between multiple messages to run, Dramatiq will run the message
 that belongs to the actor with the highest priority.
 
+.. warning::
+   Actor priority only takes effect when Dramatiq is choosing which message to run.
+   This only happens *after* messages are consumed from the queues.
+   i.e. The actory priority option has **no effect on messages while they are waiting in a queue**.
```

Comment:
Punctuation and typo: […] the queues, i.e. the actor […]

Reply:
Thanks, I have fixed by removing the `i.e.`, to keep the sentences not too long.